### PR TITLE
refactor: reset css property value when ''" (empty string) is set

### PR DESCRIPTION
**Pull request by sis0k0**

### CODE REVIEW

**Comment by sis0k0** [comment]
ios-textbase

**Comment by sis0k0** [comment]
ios-pickers

**Comment by sis0k0** [comment]
uitests

**Comment by sis0k0** [comment]
cuteness

**Comment by sis0k0** [comment]
testsappng

**Comment by sis0k0** [comment]
animations

File: tns-core-modules/ui/core/properties/properties.ts [hunk 1]
```diff
@@ -1049,6 +1050,9 @@ export class ShorthandProperty<T extends Style, P> implements definitions.Shorth
     }
 }
 
+const shouldResetCssProperty = value =>
```
Comment (PanayotCankov) [inline tns-core-modules/ui/core/properties/properties.ts]: Inline this.

File: tns-core-modules/ui/core/properties/properties.ts [hunk 2]
```diff
@@ -534,7 +535,8 @@ export class CssProperty<T extends Style, U> implements definitions.CssProperty<
         }
 
         function setCssValue(this: T, value: U): void {
-            const reset = value === unsetValue;
+            const reset = value === unsetValue ||
+                typeof value === "string" && value === "";
```
Comment (PanayotCankov) [inline tns-core-modules/ui/core/properties/properties.ts]: value === "" should suffice

**Comment by sis0k0** [comment]
uitests

**Comment by sis0k0** [comment]
cuteness

**Comment by sis0k0** [comment]
animations

File: tns-core-modules/ui/core/properties/properties.ts (hunk 2 above)
Comment (sis0k0) [reply tns-core-modules/ui/core/properties/properties.ts]: value is of type U. I can't be even explicitly casted to <string>. Should I cast it to any?

**Comment by sis0k0** [comment]
run ci

**Comment by SvetoslavTsenov** [comment]
uitests

**Pull request merged**